src/repl.zig

```diff
@@ -387,10 +407,16 @@ pub const Parser = struct {
             parser.offset = after_whitespace;
             try parser.print_current_position();
             try parser.printer.print_error(
-                \\Operation must be help, create_accounts, lookup_accounts,
-                \\create_transfers, or lookup_transfers. Got: '{s}'.
-                \\
-            ,
+                "Operation must be " ++
```

Comment:
```suggestion
                "Operation must be one of " ++
```

we are dropping `or`, so let's add `one of` so that it still reads naturally. Or maybe do

```
|operation, index|

const separator = if (index == 0) " " else if (index == Operation.len - 1) "or " else ","
```

Reply:
Using [Oxford commas](https://qph.cf2.quoracdn.net/main-qimg-790a26b47136cf4e21994999e1e32b95.webp) :smile_cat: